### PR TITLE
Update error text; adding a link to some helpful documentation.

### PR DESCRIPTION
**Pull request by ghost**
Reference: https://github.com/dotnet/cli/issues/8474

Improving the "cannot-find-the-SDK" error experience; including this link from @leecow : https://github.com/dotnet/core/blob/master/os-lifecycle-policy.md

### CODE REVIEW

**Comment by johnbeisner** [comment]
@dotnet/dotnet-cli 
Can I get a review on this please.

File: scripts/obtain/dotnet-install.ps1 [hunk 1]
```diff
@@ -565,20 +567,30 @@ catch {
...
+        }
     }
     else {
-        throw "Could not download $assetName version $SpecificVersion"
+        $DownloadFailed = $true
     }
 }
 
+if ($DownloadFailed) {
+    throw "Could not find/download: `"$assetName`" with version = $SpecificVersion`nRefer to: https://github.com/dotnet/core/blob/master/os-lifecycle-policy.md for information on .NET Core support"
```
Comment (KathleenDollard) [inline scripts/obtain/dotnet-install.ps1]: Why this URL? 

@mairaw can we get a better page in. docs and make a mslink to it, pointing temporarily at this location until the new page is ready? 

Regardless of whether we do that, should this be a direct link to GitHub or an mslink?

**Review by KathleenDollard — COMMENTED** [review]
The text looks great, I'm a bit concerned about the link because the support policy doesn't seem actionable. Happy to hear logic about why this is a good place to send people. @mairaw

File: scripts/obtain/dotnet-install.ps1 (hunk 1 above)
Comment (johnbeisner) [reply scripts/obtain/dotnet-install.ps1]: @KathleenDollard 
@leecow provided the URL - if you have a preferred URL; let's discuss.

Comment (leecow) [reply scripts/obtain/dotnet-install.ps1]: I have created https://aka.ms/dotnet-os-lifecycle, which points to the github page.

Comment (KathleenDollard) [reply scripts/obtain/dotnet-install.ps1]: I think my comment wasn't clear. The download fails: we send them to an LTS page. Hmmm. 

@johnbeisner could we brainstorm a list of reasons we'd fail here: network down? our site is down? We've removed a download? Anything else? 

@mairaw I think a page that lists the possibilities and next steps for the users would be more helpful. At the very least, I want this to be a redirect so we aren't hardcoding to the page that feels wrong. Does a "Why your .NET Core download may have failed" page belong in docs?

**Comment by johnbeisner** [comment]
@dotnet/dotnet-cli 
@KathleenDollard 
@leecow 
@mairaw 
Can someone approve these changes...

**Review by leecow — APPROVED** [review]
LGTM

**Pull request merged**